content/blog/2019/08/2019-08-13-remoting-kafka-kubernetes-release-2.0.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin release 2.0"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin release 2.0
+
+I am Long Nguyen from FPT University, Vietnam. My project for Google Summer of Code 2019 is link:https://jenkins.io/projects/gsoc/2019/remoting-over-apache-kafka-docker-k8s-features/[Remoting over Apache Kafka with Kubernetes features]. After a successful link:https://jenkins.io/blog/2019/07/11/remoting-kafka-kubernetes-phase-1/[Phase 1], finally the 2.0 version of the plugin has been released in Phase 2. The 2.0 version provides seamless integration with Kubernetes environment.
+
+== 2.0 version features
+
+* Start a simple Kafka server in Kubernetes.
+
+* Dynamically provision Remoting Kafka Agent in Kubernetes.
+
+* Helm chart to bootstrap the whole system in Kubernetes.
+
+== Start a simple Kafka server in Kubernetes
+
+Using the plugin requires users to have a configured Zookeeper and Kafka server, which would be intimidating for people who just want to try out the plugin. Now, users can start a simple, single-node Kafka server in Kubernetes environment with just one button click.
```

Comment:
"Using the plugin requires users to have" -> "Use of the plugin requires that users have"...

Reply:
"which would" -> "which could"